controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java

```diff
@@ -699,8 +699,8 @@ private void appendAndStoreCertificate(ZipBuilder zipBuilder, RunId id) {
         int testMemoryMb = (int) (1024 * (resources.memoryGb() - jdiscMemoryGb) / 2);
 
         String resourceString = String.format(Locale.ENGLISH,
-                                              "<resources vcpu=\"%.2f\" memory=\"%.2fGb\" disk=\"%.2fGb\" disk-speed=\"%s\"/>",
-                                              resources.vcpu(), resources.memoryGb(), resources.diskGb(), resources.diskSpeed().name());
+                                              "<resources vcpu=\"%.2f\" memory=\"%.2fGb\" disk=\"%.2fGb\" disk-speed=\"%s\" storage-type=\"%s\"/>",
```

Comment:
Probably not a big deal, but this will cause tester deployments to fail in the time between the controllers have upgraded until the configservers in the last zone have.

Reply:
Good point. I'll just do this one later.